File: collie/src/main/java/com/snail/collie/mem/AppMemory.java
package com.snail.collie.mem;

import android.os.Debug;

public class AppMemory {


    public long dalvikPss;//java占用内存大小
    public long nativePss;//前进程总私有已用内存大小
    public long totalPss;//当前进程总内存大小

    //  整体信息
    public Debug.MemoryInfo mMemoryInfo;
}

File: collie/src/main/java/com/snail/collie/mem/AppMemory.kt
package com.snail.collie.mem

import android.os.Debug

class AppMemory {
    @JvmField
    var dalvikPss //java占用内存大小
            : Long = 0
    @JvmField
    var nativePss //前进程总私有已用内存大小
            : Long = 0
    @JvmField
    var totalPss //当前进程总内存大小
            : Long = 0

    //  整体信息
    @JvmField
    var mMemoryInfo: Debug.MemoryInfo? = null
}